cvat/apps/engine/views.py

```diff
@@ -2924,7 +2925,16 @@ def preview(self, request: ExtendedRequest, pk: int):
 
     @extend_schema(summary='Get the status of a cloud storage',
         responses={
-            '200': OpenApiResponse(response=OpenApiTypes.STR, description='Cloud Storage status (AVAILABLE | NOT_FOUND | FORBIDDEN)'),
+            '200': OpenApiResponse(
+                # https://swagger.io/specification/#appendix-b-data-type-conversion
+                response={
+                    'type': 'string',
+                    'enum': CloudStorageStatus.list(),
+                },
+                description='Cloud Storage status'
+            ),
+            '400': OpenApiResponse(description='Outdated bucket configuration'),
```

Comment:
I don't think it makes sense to return 400 in this case. There's nothing wrong with the request - it's the bucket which is broken. Seems like we should return `NOT_FOUND` or perhaps a new type of status.